teckers/teckers/Alert.swift

```diff
@@ -21,11 +21,25 @@ class Alert {
     }
     
     func show() -> UIAlertController {
-        let alert = UIAlertController(title: self.title, message: self.massage, preferredStyle: UIAlertController.Style.alert)
+        var alert = UIAlertController()
         
         switch self.type {
         case 0:
+            alert = UIAlertController(title: self.title, message: self.massage, preferredStyle: UIAlertController.Style.alert)
+            
             alert.addAction(UIAlertAction(title: "OK", style: UIAlertAction.Style.default, handler: nil))
+        case 1:
+            alert = UIAlertController(title: self.title, message: self.massage, preferredStyle: .actionSheet)
+
+            alert.addAction(UIAlertAction(title: "Si", style: .default , handler: { (UIAlertAction) in
+                print("1")
```

Comment:
are the print() sentences necessary or they're just a test implementation?

Reply:
Just a test implementation